software/control/gui_hcs.py

```diff
@@ -742,6 +706,160 @@ def makeConnections(self):
 
         self.camera.set_callback(self.streamHandler.on_new_frame)
 
+    def makeNapariConnections(self):
```

Comment:
This is better than having them spread throughout, but still a bit confusing since it initializes non-Napari stuff when we aren't using napari for the live view.

We can go with it for now.